src/platform/src/Bridge/Meta/Llama.php

```diff
@@ -38,13 +38,17 @@ class Llama extends Model
     /**
      * @param array<string, mixed> $options
      */
-    public function __construct(string $name = self::V3_1_405B_INSTRUCT, array $options = [])
+    public function __construct(string $name = self::V3_1_405B_INSTRUCT, array $options = [], bool $supportsTools = false)
     {
         $capabilities = [
             Capability::INPUT_MESSAGES,
             Capability::OUTPUT_TEXT,
         ];
 
+        if ($supportsTools) {
```

Comment:
what do you think about:
```suggestion
        if (str_starts_with($name, 'llama-3.1') || str_starts_with($name, 'llama-3.2') || str_starts_with($name, 'llama-3.3')) {
```
could be a regex as well, yup

Reply:
Have a look at the new Ollama class. I made it a separate commit for review purposes